### DISCUSSION TITLE
CronJob manual trigger functionality

### DISCUSSION THREAD

**Discussion by dimitriospafos** [kubernetes-client/csharp#1597] [Q&A]
Hi,

I would like to know if it's possible to trigger an existing cron job ad-hoc style.
e.g: 
In the k8 dashboard: 
![k8 dash](https://github.com/user-attachments/assets/e76c7230-4212-4066-a401-0d115542e6b4)

Using native command: "kubectl create job --from=cronjob/<cronjob-name> <job-name> -n <namespace-name>"

I've used "CreateNamespacedCronJobAsync" to create a cron job. But I don't see a "from" argument that I could use to trigger the existing job as in the native command above.

Thank you

**Comment by tg123** [comment]
it is actually copy the job spec using `get` and run a new job (non cron) internally

**Reply by dimitriospafos** [reply]
What do you mean internally? is there a method i can use from the csharp sdk to run it ? Or just create a normal job that will run once
